commands/command_clone.go

```diff
@@ -85,18 +85,24 @@ func cloneCommand(cmd *cobra.Command, args []string) {
 		remote = "origin"
 	}
 
-	includeArg, excludeArg := getIncludeExcludeArgs(cmd)
-	filter := buildFilepathFilter(cfg, includeArg, excludeArg)
-	if cloneFlags.NoCheckout || cloneFlags.Bare {
-		// If --no-checkout or --bare then we shouldn't check out, just fetch instead
-		cfg.CurrentRemote = remote
-		fetchRef("HEAD", filter)
-	} else {
-		pull(remote, filter)
-		err := postCloneSubmodules(args)
-		if err != nil {
-			Exit("Error performing 'git lfs pull' for submodules: %v", err)
+	ref, err := git.CurrentRef()
+
+	if ref != nil {
```

Comment:
It's a common Go idiom to check for `err == nil` instead of `ref != nil`. You could even use `if ref, err := git.CurrentRef(); err == nil {`.